sync/build.gradle

```diff
@@ -27,7 +27,6 @@ dependencies {
     integrationTestImplementation testFixtures(project(':util'))
 
     integrationTestImplementation 'org.mockito:mockito-core'
-//    integrationTestImplementation 'org.hyperledger.besu:plugin-api'
```

Comment:
Did you intend to comment this out here?

Reply:
Yes, I'm actually deleting that line I commented but forgot to delete in a previous PR.

Reply:
Should we just delete instead of commenting-out then?

Reply:
That's what I'm doing here :D

Reply:
oh - lol 😂 read that wrong